### DISCUSSION TITLE
Reconfiguring the login node

### DISCUSSION THREAD

**Discussion by fdmalone** [GoogleCloudPlatform/cluster-toolkit#2498] [Q&A] [answered]
Is it possible to reconfigure a running cluster to change the machine type of the login node (say from a n2-standard-2 to something else)?

I've noticed when reconfiguring a running cluster the ip address of the login node changes, is it possible to have this fixed so I don't have to update my known hosts when ssh-ing in?

Thanks!

**Comment by harshthakkar01** [comment, marked as answer]
Yes, you can reconfigure a running cluster to change machine type on the login node. 

For fixed ip address, have you tried `static_ip` in login node ? you can specify using https://github.com/GoogleCloudPlatform/hpc-toolkit/blob/main/community/modules/scheduler/schedmd-slurm-gcp-v6-login/variables.tf#L96.
